eo/notebooks/process_management.ipynb

```diff
@@ -0,0 +1,843 @@
+    "\n",
+    "#### Requirements\n",
+    "- Weaver >=4.30.0 \n",
+    "- Installed the weaver client : https://pavics-weaver.readthedocs.io/en/latest/installation.html\n",
+    "    - After the installation make sure this notebook has access to the weaver client conda environnement. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7564bab",
+   "metadata": {},
+   "source": [
+    "### Weaver Definitions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9658802c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from weaver.cli import WeaverClient\n",
+    "from requests_magpie import MagpieAuth\n",
+    "\n",
+    "from pprint import pprint\n",
+    "import requests\n",
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bed0a9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Weaver setup\n",
+    "WEAVER_URL = os.getenv('WEAVER_URL')\n",
+    "\n",
+    "# Authentication \n",
+    "MAGPIE_URL = os.getenv('MAGPIE_URL')\n",
+    "MAGPIE_USER = os.getenv('MAGPIE_USER')\n",
+    "MAGPIE_PASS = os.getenv('MAGPIE_PASS)\n",
+    "\n",
+    "# Magpie Authentication\n",
+    "aCMagpieAuth = MagpieAuth(MAGPIE_URL, MAGPIE_USER, MAGPIE_PASS)\n",
+    "\n",
+    "# Instanciate a weaver client with magpie authentication\n",
+    "wclient = WeaverClient(url=WEAVER_URL, auth=aCMagpieAuth)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8aa90709",
+   "metadata": {},
+   "source": [
+    "### Weaver Capabilities\n",
+    "This is a way for us to know which process is available on the machine pointed by the weaver client."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6891422f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def print_available_processes():\n",
+    "    print('=== Available Processes ===')\n",
+    "    response = wclient.capabilities()\n",
+    "    processes = response.body['processes']\n",
+    "    pprint(processes)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "id": "d5a80241",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "=== Available Processes ===\n",
+      "['file_index_selector',\n",
+      " 'file2string_array',\n",
+      " 'jsonarray2netcdf',\n",
+      " 'metalink2netcdf']\n"
+     ]
+    }
+   ],
+   "source": [
+    "print_available_processes()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e2f3b28",
+   "metadata": {},
+   "source": [
+    "## 2. Deploy the process on Weaver\n",
+    "[Deploy Documentation](https://pavics-weaver.readthedocs.io/en/latest/autoapi/weaver/cli/index.html#weaver.cli.WeaverClient.deploy)</br>\n",
+    "\n",
+    "This is the part where we send to Weaver our CWL process. In our case this would be the `sentinelsat_download_process.yml` file. \n",
```

Comment:
This file seems to be missing.

If added, again refer to "demo/sentinelsat_download_process.cwl" to better guide the location.

Reply:
Oh yea you're right this is supposed to be the .cwl